app/src/main/java/com/sh1r0/noveldroid/SettingsFragment.java

```diff
@@ -157,8 +195,7 @@ public void onResume() {
 		super.onResume();
 		getActivity().setTitle(R.string.settings);
 		encoding.setSummary(prefs.getString(KEY_ENCODING, "UTF-8"));
-		namingRulePref = Integer.parseInt(prefs.getString(KEY_NAMING_RULE, "0"));
-		namingRule.setSummary(namingRuleList[namingRulePref]);
+		namingRule.setSummary(prefs.getString(KEY_NAMING_RULE, DEFAULT_NAMING_RULE));
```

Comment:
The summary here could be the preview result of the input pattern, for example, `/a_/n => author_name`, where `/` is the prefix pattern.